### PR TITLE
Migrate Python3 transforms package.

### PR DESCRIPTION
**Pull request by tneymanov**

### CODE REVIEW

File: gcp_variant_transforms/transforms/annotate_files.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/annotate_files.py:15]: line.

File: gcp_variant_transforms/transforms/densify_variants.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 """A PTransform to extend each Variant's calls with data for all samples."""
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/densify_variants.py:17]: line.

File: gcp_variant_transforms/transforms/densify_variants_test.py [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 """Tests for densify_variants module."""
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/densify_variants_test.py:17]: line.

File: gcp_variant_transforms/transforms/extract_input_size.py [hunk 4]
```diff
@@ -22,7 +22,7 @@
   - file count: number of input files.
 """
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/extract_input_size.py:25]: line.

File: gcp_variant_transforms/transforms/filter_variants.py [hunk 5]
```diff
@@ -14,7 +14,7 @@
 
 """A PTransform for filtering variants."""
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/filter_variants.py:17]: line.

File: gcp_variant_transforms/transforms/filter_variants_test.py [hunk 6]
```diff
@@ -14,7 +14,7 @@
 
 """Tests for filter_variants module."""
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/filter_variants_test.py:17]: line.

File: gcp_variant_transforms/transforms/infer_headers.py [hunk 7]
```diff
@@ -14,7 +14,7 @@
 
 """A PTransform to infer undefined/mismatched header fields."""
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/infer_headers.py:17]: line.

File: gcp_variant_transforms/transforms/infer_headers_test.py [hunk 8]
```diff
@@ -14,7 +14,7 @@
 
 """Tests for infer_headers module."""
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/infer_headers_test.py:17]: line.

File: gcp_variant_transforms/transforms/infer_headers_test.py [hunk 9]
```diff
@@ -304,7 +304,7 @@ def test_infer_annotation_types_no_conflicts(self):
     expected_types = {'CSQ_Gene_TYPE': 'String',
                       'CSQ_Position_TYPE': 'Integer',
                       'CSQ_Score_TYPE': 'Float'}
-    for key, item in inferred_headers.infos.iteritems():
+    for key, item in list(inferred_headers.infos.items()):
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/infer_headers_test.py:307]: Remove list.

File: gcp_variant_transforms/transforms/infer_headers_test.py [hunk 10]
```diff
@@ -321,7 +321,7 @@ def test_infer_annotation_types_with_type_conflicts(self):
     expected_types = {'CSQ_Gene_TYPE': 'Float',
                       'CSQ_Position_TYPE': 'String',
                       'CSQ_Score_TYPE': 'Float'}
-    for key, item in inferred_headers.infos.iteritems():
+    for key, item in list(inferred_headers.infos.items()):
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/infer_headers_test.py:324]: Remove list.

File: gcp_variant_transforms/transforms/infer_headers_test.py [hunk 11]
```diff
@@ -338,7 +338,7 @@ def test_infer_annotation_types_with_missing(self):
     expected_types = {'CSQ_Gene_TYPE': '.',
                       'CSQ_Position_TYPE': 'Integer',
                       'CSQ_Score_TYPE': 'Float'}
-    for key, item in inferred_headers.infos.iteritems():
+    for key, item in list(inferred_headers.infos.items()):
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/infer_headers_test.py:341]: Remove list.

File: gcp_variant_transforms/transforms/infer_headers_test.py [hunk 12]
```diff
@@ -370,7 +370,7 @@ def test_infer_annotation_types_with_multiple_annotation_fields(self):
                       'CSQ_VT_Gene_TYPE': 'Integer',
                       'CSQ_VT_Position_TYPE': 'Integer',
                       'CSQ_VT_Score_TYPE': 'Float'}
-    for key, item in inferred_headers.infos.iteritems():
+    for key, item in list(inferred_headers.infos.items()):
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/infer_headers_test.py:373]: Remove list.

File: gcp_variant_transforms/transforms/limit_write.py [hunk 13]
```diff
@@ -21,7 +21,7 @@
 are BigQuery rows) based on that key before writing them to output table.
 """
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/limit_write.py:24]: line.

File: gcp_variant_transforms/transforms/limit_write_test.py [hunk 14]
```diff
@@ -14,7 +14,7 @@
 
 """Tests for limit_write module."""
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/limit_write_test.py:17]: line.

File: gcp_variant_transforms/transforms/merge_headers.py [hunk 15]
```diff
@@ -52,7 +52,7 @@ def merge_accumulators(self, accumulators):
 
   def extract_output(self, merged_headers):
     # type: (vcf_header_io.VcfHeader) -> vcf_header_io.VcfHeader
-    for info in merged_headers.infos.values():
+    for info in list(merged_headers.infos.values()):
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/merge_headers.py:55]: Remove list.

File: gcp_variant_transforms/transforms/merge_variants.py [hunk 16]
```diff
@@ -14,7 +14,7 @@
 
 """A PTransform for merging variants based on a strategy."""
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/merge_variants.py:17]: line.

File: gcp_variant_transforms/transforms/merge_variants.py [hunk 17]
```diff
@@ -42,7 +42,8 @@ def _map_by_variant_keys(self, variant):
     for key in self._variant_merger.get_merge_keys(variant):
       yield (key, variant)
 
-  def _merge_variants_by_key(self, (key, variants)):
+  def _merge_variants_by_key(self, xxx_todo_changeme):
+    (key, variants) = xxx_todo_changeme
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/merge_variants.py:46]: `xxx_todo_changeme` needs to be fixed.

File: gcp_variant_transforms/transforms/merge_variants_test.py [hunk 18]
```diff
@@ -14,7 +14,7 @@
 
 """Tests for merge_variants module."""
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/merge_variants_test.py:17]: line.

File: gcp_variant_transforms/transforms/sample_mapping_table_test.py [hunk 19]
```diff
@@ -14,7 +14,7 @@
 
 """Tests for densify_variants module."""
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/sample_mapping_table_test.py:17]: line.

File: gcp_variant_transforms/transforms/shard_variants.py [hunk 20]
```diff
@@ -14,7 +14,7 @@
 
 """A PTransform for sharding variants based on their reference_name."""
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/shard_variants.py:17]: line.

File: gcp_variant_transforms/transforms/shard_variants_test.py [hunk 21]
```diff
@@ -14,7 +14,7 @@
 
 """Tests for shard_variants module."""
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/shard_variants_test.py:17]: line.

File: gcp_variant_transforms/transforms/shard_variants_test.py [hunk 22]
```diff
@@ -85,7 +85,7 @@ def _get_expected_variant_shards(self):
   def test_shard_variants(self):
     expected_shards = self._get_expected_variant_shards()
     variants = [variant
-                for variant_list in expected_shards.values()
+                for variant_list in list(expected_shards.values())
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/shard_variants_test.py:88]: Remove list.

File: gcp_variant_transforms/transforms/variant_to_avro.py [hunk 23]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/variant_to_avro.py:15]: line.

File: gcp_variant_transforms/transforms/variant_to_bigquery.py [hunk 24]
```diff
@@ -14,7 +14,7 @@
 
 """A PTransform to output a PCollection of ``Variant`` records to BigQuery."""
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/variant_to_bigquery.py:17]: line.

File: gcp_variant_transforms/transforms/variant_to_bigquery_test.py [hunk 25]
```diff
@@ -14,7 +14,7 @@
 
 """Tests for variant_to_bigquery module."""
 
-from __future__ import absolute_import
+
```
Comment (samanvp) [inline gcp_variant_transforms/transforms/variant_to_bigquery_test.py:17]: line.

**Review by samanvp — APPROVED** [review]
Again a bunch of nits, except for `xxx_todo_changeme`. Please make sure you fix that.

File: gcp_variant_transforms/transforms/densify_variants.py (hunk 2 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/densify_variants.py:17]: Done.

File: gcp_variant_transforms/transforms/densify_variants_test.py (hunk 3 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/densify_variants_test.py:17]: Done.

File: gcp_variant_transforms/transforms/extract_input_size.py (hunk 4 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/extract_input_size.py:25]: Done.

File: gcp_variant_transforms/transforms/filter_variants.py (hunk 5 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/filter_variants.py:17]: Done.

File: gcp_variant_transforms/transforms/infer_headers.py (hunk 7 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/infer_headers.py:17]: Done.

File: gcp_variant_transforms/transforms/filter_variants_test.py (hunk 6 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/filter_variants_test.py:17]: Done.

File: gcp_variant_transforms/transforms/infer_headers_test.py (hunk 8 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/infer_headers_test.py:17]: Done.

File: gcp_variant_transforms/transforms/infer_headers_test.py (hunk 9 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/infer_headers_test.py]: Done.

File: gcp_variant_transforms/transforms/infer_headers_test.py (hunk 10 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/infer_headers_test.py]: Done.

File: gcp_variant_transforms/transforms/infer_headers_test.py (hunk 11 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/infer_headers_test.py]: Done.

File: gcp_variant_transforms/transforms/infer_headers_test.py (hunk 12 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/infer_headers_test.py]: Done.

File: gcp_variant_transforms/transforms/limit_write.py (hunk 13 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/limit_write.py:24]: Done.

File: gcp_variant_transforms/transforms/limit_write_test.py (hunk 14 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/limit_write_test.py:17]: Done.

File: gcp_variant_transforms/transforms/merge_headers.py (hunk 15 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/merge_headers.py]: Done.

File: gcp_variant_transforms/transforms/merge_variants.py (hunk 16 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/merge_variants.py:17]: Done.

File: gcp_variant_transforms/transforms/merge_variants.py (hunk 17 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/merge_variants.py]: I modified it in future PR, but I'll do it here.

File: gcp_variant_transforms/transforms/merge_variants_test.py (hunk 18 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/merge_variants_test.py:17]: Done.

File: gcp_variant_transforms/transforms/sample_mapping_table_test.py (hunk 19 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/sample_mapping_table_test.py:17]: Done.

File: gcp_variant_transforms/transforms/shard_variants.py (hunk 20 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/shard_variants.py:17]: Done.

File: gcp_variant_transforms/transforms/shard_variants_test.py (hunk 21 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/shard_variants_test.py:17]: Done.

File: gcp_variant_transforms/transforms/shard_variants_test.py (hunk 22 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/shard_variants_test.py]: Done.

File: gcp_variant_transforms/transforms/variant_to_avro.py (hunk 23 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/variant_to_avro.py:15]: Done.

File: gcp_variant_transforms/transforms/variant_to_bigquery.py (hunk 24 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/variant_to_bigquery.py:17]: Done.

File: gcp_variant_transforms/transforms/variant_to_bigquery_test.py (hunk 25 above)
Comment (tneymanov) [reply gcp_variant_transforms/transforms/variant_to_bigquery_test.py:17]: Done.

**Pull request merged**